emails/migrations/0020_reply_created_at.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 2.2.24 on 2021-08-31 20:29
+
+from django.db import migrations, models
+import django.utils.timezone
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('emails', '0019_merge_20210825_1737'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='reply',
+            name='created_at',
+            field=models.DateField(auto_now_add=True, default=django.utils.timezone.now),
+            preserve_default=False,
```

Comment:
We want to verify that the date of the Reply object created is properly populated.

Reply:
I verified the reply records all have the proper (i.e., distinct) values.
![image](https://user-images.githubusercontent.com/71928/134207813-46e02ebe-c5a9-46da-9244-d5361cded2d8.png)